pdf-service/app/main.py

```
@@ -1,16 +1,52 @@
+from .adapters.aup_protocol import from_working_content as aup_from_working_content
+from .adapters.vet_patrol import from_animals as vet_patrol_from_animals
 from .auth import require_internal_token
-from .doc_types import DOCX_REGISTRY, REGISTRY
+from .doc_types import DOCX_REGISTRY, REGISTRY, XLSX_REGISTRY
 from .docx_renderer import DOCX_MIMETYPE, docx_to_pdf, render_docx
 from .renderer import gotenberg_healthy, html_to_pdf, render_html
+from .xlsx_renderer import XLSX_MIMETYPE, render_xlsx
 
 logger = logging.getLogger(__name__)
 
+# R32-A3 收尾：Word COM converter（host daemon）→ 100% Word-fidelity PDF
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)